test/reporter/reporter.go

```diff
@@ -141,6 +142,7 @@ func (r *KubernetesNMStateReporter) logPods(testName string, sinceTime time.Time
 }
 
 func (r *KubernetesNMStateReporter) OpenTestLogFile(logType string, testName string, cb func(f io.Writer), extraWriters ...io.Writer) {
+	syscall.Umask(0)
```

Comment:
why do we need this? Shouldn't we change it back to the original mask?

Reply:
Removed, it was an experiment.